src/components/SideBar/SideBar.vue

```diff
@@ -81,6 +82,16 @@ export default {
     },
     showLogout() {
       return this.$store.getters['app/config'].sidebar.logout
+    },
+    showSideBarFooterComponent() {
+      if (this.$store.getters['app/config'].sidebar.customSideBarFooterComponent) {
+        return this.$store.getters['app/config'].sidebar.customSideBarFooterComponent.enabled
+      }
+    },
+    sideBarFooterComponent() {
```

Comment:
Might as well add the header component while you're adding this one

Reply:
The current optional sidebar user profile should use the new sidebar header slot by default (if configured to be enabled)